File: Facebook.NET - VB/Contracts/GraphAPI.Settings.vb
Imports System
Imports System.Linq
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class Settings
            <DataMember(Name:="enabled")>
            Public Property Enabled() As System.Nullable(Of Boolean)
        End Class
    End Class
End Namespace

File: Facebook.NET/Contracts/GraphAPI.Settings.cs
using System;
using System.Linq;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public sealed class Settings
        {
            [DataMember(Name = "enabled")]
            public bool? Enabled { get; set; }
        }
    }
}